src/main/scala/cromwell/binding/WdlExpression.scala

```diff
@@ -14,6 +13,34 @@ import scala.util.{Failure, Success, Try}
 class WdlExpressionException(message: String = null, cause: Throwable = null) extends RuntimeException(message, cause)
 
 object WdlExpression {
+
+  implicit class AstForExpressions(val ast: Ast) extends AnyVal {
+    def isFunctionCall: Boolean = ast.getName == "FunctionCall"
+    def isBinaryOperator: Boolean = binaryOperators.contains(ast.getName)
+    def isUnaryOperator: Boolean = UnaryOperators.contains(ast.getName)
+    def functionName: String = ast.getAttribute("name").asInstanceOf[Terminal].getSourceString
```

Comment:
Maybe some other things might want to be implicited:

`ast.getAttribute("params").asInstanceOf[AstList].asScala.toVector`
`PreEvaluableFunctions_SingleParameter.contains(ast.functionName))`
`ast.isFunctionCall && AllowedInnerFunctionCalls.contains(ast.functionName)`

etc